<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowReachability</class>
 <widget class="QMainWindow" name="MainWindowReachability">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>868</width>
    <height>707</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string> VirtualRobot - ReachabilityMap Demo</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QFrame" name="frameViewer">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBox">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>460</y>
         <width>201</width>
         <height>271</height>
        </rect>
       </property>
       <property name="title">
        <string>Visualization</string>
       </property>
       <widget class="QCheckBox" name="checkBoxRobot">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>70</y>
          <width>131</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Robot</string>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBoxReachabilityVisu">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>90</y>
          <width>131</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Reachability Map</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBoxObject">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>131</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Object</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QCheckBox" name="checkBoxGrasps">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>131</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Grasp(s)</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>70</y>
         <width>201</width>
         <height>261</height>
        </rect>
       </property>
       <property name="title">
        <string>Reachability Map</string>
       </property>
       <widget class="QRadioButton" name="radioButtonAllGrasps">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>120</y>
          <width>82</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>All Grasps</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButtonOneGrasp">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>140</y>
          <width>82</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>One Grasp</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBoxGrasp">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>160</y>
          <width>131</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>121</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>End Effector</string>
        </property>
       </widget>
       <widget class="QComboBox" name="comboBoxEEF">
        <property name="geometry">
         <rect>
          <x>18</x>
          <y>60</y>
          <width>151</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QPushButton" name="pushButtonObjectRandom">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>Place Object Random</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>868</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>radioButtonAllGrasps</tabstop>
  <tabstop>radioButtonOneGrasp</tabstop>
  <tabstop>comboBoxGrasp</tabstop>
  <tabstop>checkBoxObject</tabstop>
  <tabstop>checkBoxGrasps</tabstop>
  <tabstop>checkBoxRobot</tabstop>
  <tabstop>checkBoxReachabilityVisu</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
